packages/sprotty-vscode/src/lsp/lsp-webview-endpoint.ts

```diff
@@ -40,25 +40,29 @@ export class LspWebviewEndpoint extends WebviewEndpoint {
 
     protected override connect(): void {
         super.connect();
-        this.messenger.onRequest(LspRequest,
-            async request => {
-                const result: any = request.params === undefined
-                    ? await this.languageClient.sendRequest(request.method)
-                    : await this.languageClient.sendRequest(request.method, request.params);
-                const response: ResponseMessage = {
-                    jsonrpc: '2.0',
-                    id: request.id,
-                    result
-                };
-                return response;
-            },
-            { sender: this.messageParticipant }
+        super.disposables.push(
```

Comment:
I get `undefined` runtime errors here. Try changing to `this.disposables`

Reply:
Fixed, thanks!